src/appshell/view/navigableappmenumodel.cpp

```diff
@@ -221,6 +221,9 @@ bool NavigableAppMenuModel::processEventForOpenedMenu(QEvent* event)
 
 bool NavigableAppMenuModel::processEventForAppMenu(QEvent* event)
 {
+    if (event->type() == QEvent::Type::MouseButtonPress) {
+        m_needActivateHighlight = false;
```

Comment:
this is necessary as otherwise after using ALT+drag the keyboard focus switches to the menu